src/router/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { HomePage } from '@/pages/Home'
 import { AboutPage } from '@/pages/About'
+import { LoginPage } from '@/pages/LoginPage'
```

Comment:
El nombre del archivo sería Login
```suggestion
import { LoginPage } from '@/pages/Login'
```